_layouts/frontpage.html

```diff
@@ -157,40 +157,34 @@ <h2>Projects that we've worked on</h2>
         <div class="vertical-spacer-top" id="our-projects"></div>
         <div class="open-source">
             <div class="container">
-                <h2>Currently working on...</h2>
-                <div class="projects">
-                    {% for project in site.projects %}
-                      {% if project.hide-from-homepage != true %}
-                      <a href="/projects.html#{{ project.label }}" class="project">
-                        {% case project.status %}
-                          {% when "Contributors Welcome!" %}
-                            {% assign projectStatus ="contributorswelcome" %}
-                          {% else %}
-                            {% assign projectStatus = project.status %}
-                        {% endcase %}
-                        <div class="status {{ project.status| downcase | remove: " " | remove: "!" }}">{{ project.status }}</div>
-                        <div class="icon-status {{ projectStatus|downcase }}">
-                          <i class="fa fa-{{ projectStatus|downcase }}"></i>
-                        </div>
-                        {% if project.logo-home %}
-                        <div class="logo"><img src="{{ project.logo-home }}" alt="{{ project.name }}"></div>
-                        {% endif %}
-                        <div class="text">
-                          {% if project.short-name %}
-                            {{ project.short-name }}
-                          {% else %}
-                            {{ project.name }}
-                          {% endif %}
-                        </div>
-                      </a>
-                      {% endif %}
-                    {% endfor %}
-                </div>
-
+                <h2>Current priorities</h2>
                 <div class="clearfix"></div>
+                <div class="text-left">
+                  <div>
+                    <ul class="red">
+                      <li>
+                        <strong>Communicate excitement about Scala</strong>. We want more people (programmers, teachers,
+                        scientists) to know about Scala’s strengths.
+                      </li>
+                      <li>
+                        <strong>Newcomers onboarding experience</strong>. We want newcomers to quickly get started in
+                        Scala, with no hassles.
```

Comment:
```suggestion
                        Scala, with no hassle.
```

Reply:
I think “no hassles” is correct? Maybe Jamie or Sébastien can help?

Reply:
@bishabosha